src/util/Handlers.js

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define(['util/Logger'], function (Logger) {
+
+  var Handlers = {};
+
+  Handlers.defaultSuccessTokenHandler = function(tokenManager) {
+    /**
+     * Default success handler which is normally invoked when
+     * `parseTokenFromUrl` is called. This success handler will
+     * automatically store the tokens with the keys "accessToken" and
+     * "idToken", if returned in the response.
+     */
+    return function (res) {
+      var tokens = Array.isArray(res) ? res : [res];
+      for (var i = 0; i < tokens.length; ++i) {
```

Comment:
Perhaps personal pref, but`tokens.forEach(token => {})` feels more straightforward?

Reply:
This was mostly a speed decision. Storing the tokens already takes a bit of time due to their size, so I wanted to try and keep performance up as high as possible.

```
// Store accessToken and idToken

// foreEach
33752.566650390625ms

// for loop
20372.998779296875ms
```

Reply:
Tell me what those numbers are telling me/ how you got them? There are performance tradeoffs with foreach vs for, but I'd be surprised if a two element array, evaluated once per app instance, causes any significant impact? This minimal use case is why I was suggesting a readability improvement over a possible performance issue.

Reply:
The numbers above are execution times for storing an `accessToken` and `idToken` into `localStorage`. The first, uses the `forEach` method as a replacement for the line in question [here](https://github.com/okta/okta-signin-widget/pull/494/files/53f106e9feaf0fc0ebd5bcef5c6932d9bdabd6fb#diff-e86f1699ebb182f99bfd831860aeeb29R26), while the second is the current state.

Both calls were measured using `console.time` and `console.timeEnd`.

Since we're looking at a `40%` time difference between the two, it makes the most sense to me to choose the basic loop - especially since this is happening during a state transition.